src/main/java/com/books/servlets/LoginServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.books.servlets;
+
+import com.books.entities.User;
+import com.books.services.AuthenticateService;
+import com.books.services.EncryptService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+import java.io.IOException;
+
+@WebServlet("/LoginServlet")
```

Comment:
Why you have this annotation, if you already have web.xml descriptor. Please do not use code constructions without pure understanding.